tests/models/mask2former/test_image_processing_mask2former.py

```diff
@@ -242,6 +269,15 @@ def common(is_instance_map=False, segmentation_type=None):
         common(is_instance_map=True)
         common(is_instance_map=False, segmentation_type="pil")
         common(is_instance_map=True, segmentation_type="pil")
+        common(num_channels=1, numpify=True)
+        common(num_channels=2, numpify=True, input_data_format=ChannelDimension.LAST)
+        common(num_channels=5, numpify=True, input_data_format=ChannelDimension.LAST, do_resize=False)
+
+        with pytest.raises(ValueError, match="Unable to infer channel dimension format"):
+            common(num_channels=5, numpify=True, do_resize=False)
+
+        with pytest.raises(TypeError, match=r"Cannot handle this data type: .*"):
+            common(num_channels=5, numpify=True, input_data_format=ChannelDimension.LAST)
```

Comment:
Sorry, missed this for the first time, we use unittest for testing, can you please update to `with self.assertRaises(ValueError)` or `assertRaisesRegex` and remove `import pytest` on the top of the module.